adapters/WorldGuardAdapter/src/main/kotlin/com/caleb/typewriter/worldguard/entries/fact/InRegionFact.kt

```diff
@@ -11,7 +11,7 @@ import me.gabber235.typewriter.facts.Fact
 import me.gabber235.typewriter.utils.Icons
 import java.util.*
 
-@Entry("in_region_fact", "If the player is in a region", Colors.PURPLE, Icons.ROAD_BARRIER)
+@Entry("in_wg_region_fact", "If the player is in a WorldGuard region", Colors.PURPLE, Icons.ROAD_BARRIER)
```

Comment:
Though I do agree with this change, don't include it in this pr. Such a changes requires me to create a migration flow as users are possibly already using the this entry and making this update will break their pages. 

Maybe create a separate pr for it.